app/models/global_issue_template.rb

```diff
@@ -39,4 +39,17 @@ def enabled?
   def <=>(global_issue_template)
     position <=> global_issue_template.position
   end
+
+  #
+  # Class method
+  #
+  class << self
+    def get_templates_for_project_tracker(project_id, tracker_id=nil)
```

Comment:
__[rubocop]__
* Surrounding space missing in default value assignment. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-around-equals)